components/crypto_dot_com/browser/crypto_dot_com_json_parser.h

```diff
@@ -0,0 +1,32 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_CRYPTO_DOT_COM_BROWSER_CRYPTO_DOT_COM_JSON_PARSER_H_
+#define BRAVE_COMPONENTS_CRYPTO_DOT_COM_BROWSER_CRYPTO_DOT_COM_JSON_PARSER_H_
+
+#include <map>
+#include <string>
+#include <vector>
+
+#include "brave/components/crypto_dot_com/browser/crypto_dot_com_service.h"
```

Comment:
Include seems unused here

Reply:
type definitions are grabbed from this file

Reply:
nit: for encapsulation of concerns, I think in a follow up it'd be good to not include the service in this area of code. Not a big deal though.